studio/lib/constants/metrics.tsx

```diff
@@ -416,16 +416,17 @@ export const TIME_PERIODS_INFRA = [
   {
     key: '1h',
     label: 'Last hour',
-    interval: '1m',
```

Comment:
I removed these as, these values seems to be overwritten here anyways:
https://github.com/supabase/supabase/blob/master/studio/components/to-be-cleaned/DateRangePicker.tsx#L67-L118